.github/workflows/test_destinations_local.yml

```diff
@@ -4,40 +4,76 @@
 name: dest | postgres, duckdb, fs, weaviate, qdrant
 
 on:
-  pull_request:
-    branches:
-      - master
-      - devel
+  workflow_call:
   workflow_dispatch:
 
-concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
-  cancel-in-progress: true
-
 env:
-  # NOTE: this workflow can't use github secrets!
-  # DLT_SECRETS_TOML: ${{ secrets.DLT_SECRETS_TOML }}
 
   # RUNTIME__SENTRY_DSN: https://6f6f7b6f8e0f458a89be4187603b55fe@o1061158.ingest.sentry.io/4504819859914752
   RUNTIME__LOG_LEVEL: ERROR
   RUNTIME__DLTHUB_TELEMETRY_ENDPOINT: ${{ secrets.RUNTIME__DLTHUB_TELEMETRY_ENDPOINT }}
-  ACTIVE_DESTINATIONS: "[\"duckdb\", \"postgres\", \"filesystem\", \"weaviate\", \"qdrant\"]"
-  ALL_FILESYSTEM_DRIVERS: "[\"memory\", \"file\", \"sftp\"]"
-
-  DESTINATION__WEAVIATE__VECTORIZER: text2vec-contextionary
-  DESTINATION__WEAVIATE__MODULE_CONFIG: "{\"text2vec-contextionary\": {\"vectorizeClassName\": false, \"vectorizePropertyName\": true}}"
 
 jobs:
-  get_docs_changes:
-    name: docs changes
-    uses: ./.github/workflows/get_docs_changes.yml
-
-  run_loader:
-    name: dest | postgres, duckdb and fs local tests
-    needs: get_docs_changes
-    if: needs.get_docs_changes.outputs.changes_outside_docs == 'true'
+
+  run_destinations_local:
+    name: dest | local
+
     strategy:
       fail-fast: false
+      matrix:
+        include:
+
+          # Duckdb, Postgres, Filesystem, Weaviate, Qdrant (could be split?)
+          - name: filesystem, weaviate, qdrant
+            destinations: "[\"filesystem\", \"weaviate\", \"qdrant\"]"
+            filesystem_drivers: "[\"memory\", \"file\", \"sftp\"]"
+            extras: "parquet cli filesystem qdrant weaviate deltalake pyiceberg"
+            needs_weaviate: true
+            needs_qdrant: true
+            needs_ftp: true
+            post_install_commands: "poetry run pip install sqlalchemy==2.0.18" # minimum version required by `pyiceberg`
+            additional_tests: "poetry run pytest tests/cli"
+
+          - name: postgres, duckdb
+            destinations: "[\"postgres\", \"duckdb\"]"
+            filesystem_drivers: "[\"memory\", \"file\"]"
+            extras: "postgres postgis parquet duckdb cli filesystem"
+            needs_postgres: true
+
+          # Clickhouse OSS (TODO: test with minio s3)
```

Comment:
clickhouse oss is disabled for now, it would be good to make it work without any remote services, for example test it with s3 using minio

Reply:
I think it uses the local clickhouse but depends on s3, az etc. are you sure you can run the full test suite without network connection? I will uncomment it again and have another look tomorrow, but I think it will not work without secrets.